core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2547,7 +2547,7 @@ class SparkContext(config: SparkConf) extends Logging {
   /**
    * Cancel active jobs that have the specified tag. See `org.apache.spark.SparkContext.addJobTag`.
    *
-   * @param tag The tag to be added. Cannot contain ',' (comma) character.
+   * @param tag The tag to be cancelled. Cannot contain ',' (comma) character.
```

Comment:
piggy back this typy that I noticed

Reply:
Mind fixing `JavaSparkContext.cancelJobsWithTag` too since we're here? I just copied and pasted it 😂